bin/tass

```diff
@@ -109,6 +109,19 @@ when 'create'
         end
       end
     end
+
+    if config.key?(:asg_metrics)
+      # We list all metrics because otherwise we can't get GroupStandbyInstances
+      Tapjoy::AutoscalingBootstrap::AWS::Autoscaling.client.enable_metrics_collection({
+        auto_scaling_group_name: Tapjoy::AutoscalingBootstrap.scaler_name,
+        metrics: %w(
```

Comment:
But @ehealy that would be something a reasonable person would design.

From the SDK docs:
> Note that the GroupStandbyInstances metric is not enabled by default. You must explicitly request this metric.